text/0813-template-layering.md

```diff
@@ -30,9 +30,10 @@ export const Foo = template(`<Hello />`, () => ({ Hello }));
 // export class Bar {
 //   <template><Hello /></template>
 // }
-export
-@template(`<Hello />`, () => ({ Hello })
-class Bar {
+export class Bar {
+  static {
```

Comment:
I was initially against this, but I worked through it here
https://github.com/emberjs/rfcs/pull/813#discussion_r851770215

and now I'm hugely in favor :100: